charts/rancher-backup/README.md

```diff
@@ -42,6 +42,8 @@ The following table lists the configurable parameters of the rancher-backup char
 | persistence.storageClass |  StorageClass to use for dynamically provisioning the Persistent Volume, which will be used for storing backups | "" |
 | persistence.volumeName |  Persistent Volume to use for storing backups | "" |
 | persistence.size |  Requested size of the Persistent Volume (Applicable when using dynamic provisioning) | "" |
+| debug | Set debug flag for backup-restore deployment | false |
```

Comment:
Can we change this to `Enable debug logging for operator` and `Enable trace logging for operator (this can contain sensitive info)`